### DISCUSSION TITLE
my college does not provide email id

### DISCUSSION THREAD

**Discussion by bhagirath1312** [community/community#61269] [GitHub Education]
### Hi everyone,

My GitHub student pack is expiring soon, and I try to re-verify, but I don't have a collage email ID, so I use my own email ID, and I upload my document, but it is rejected. I tried 12 times.

**Comment by samus-aran** [comment]
Hi @bhagirath1312  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
